terraform/shared/modules/env/logshipper.tf

```diff
@@ -0,0 +1,24 @@
+locals {
+  logshipper_name = "logshipper"
+}
+
+module "cg-logshipper" {
+  source               = "../cg-logshipper"
+  name                 = local.logshipper_name
+  cf_org_name          = var.cf_org_name   # gsa-tts-oros-fac
+  cf_space_name        = var.cf_space_name # eg prod
+  https_proxy          = module.https-proxy.https_proxy
+  logshipper_instances = 1
+  logshipper_memory    = 1046
```

Comment:
What's with the weird number...? Also we're using FluentBit because it requires so little memory generally; do we really need this much dedicated here?

Reply:
Oh, this was a test (i think) to see if I could resolve the 502 issue by giving it more memory, but yea, good catch, bumping it back down to 256